source/blog/2017-07-31-fedora-atomic-25-to-26-upgrade.md

```diff
@@ -0,0 +1,274 @@
+---
+title: 'Fedora 25->26 Atomic Host Upgrade Guide'
+author: dustymabe
+date: 2017-07-31
+tags: fedora, atomic
+published: false
+---
+
+
+# Introduction
+
+Earlier this month we put out the
+[first](http://www.projectatomic.io/blog/2017/07/fedora-atomic-26-release/) 
+and
+[second](http://www.projectatomic.io/blog/2017/07/fedora-atomic-july-25/)
+releases of Fedora 26 Atomic Host. In this blog post we'll cover
+updating an existing Fedora 25 Atomic Host system to Fedora 26.
+We'll cover preparing the system for upgrade and performing the upgrade.
+
+**NOTE:** If you really don't want to upgrade to Fedora 26 see the
+          later section: *Appendix B: Fedora 25 Atomic Host Life Support*.
+
+# Preparing for Upgrade
+
+Before we update to Fedora 26 Atomic Host we should check to
+see that we have at least a few GiB of free space in our root
+filesystem. The update to Fedora 26 can cause your system to 
+retrieve more than 1GiB of new content (not shared with Fedora
+25) and thus we'll need to make sure we have plenty of free space.
+
+**NOTE:** Upstream OSTree has implemented some
+          [filesystem checks](https://github.com/ostreedev/ostree/pull/987) 
+          to make sure that upgrades will stop themselves before filling up the
+          filesystem and possibly corrupting your system. 
```

Comment:
Then isn't telling someone that the upgrade will stop itself rather deceptive?  If it won't stop itself while they are upgrading from F25 to F26?